### DISCUSSION TITLE
3.0版本，line折线图颜色无法修改

### DISCUSSION THREAD

**Discussion by XzavierDion** [XCharts-Team/XCharts#251] [Q&A] [answered]
3.0版本，line折线图，我设置了line style > color为红色, 左边颜色无法显示
<img width="365" alt="截屏2023-03-24 14 24 02" src="https://user-images.githubusercontent.com/34223067/227443372-d000fd1a-d8de-4cb3-acd1-72ac241f59a7.png">
<img width="1221" alt="截屏2023-03-24 14 30 07" src="https://user-images.githubusercontent.com/34223067/227443381-86391f8c-4289-47ba-8de3-4848b58692c8.png">

**Comment by monitor1394** [comment, marked as answer]
注意颜色的a值
